net-wireless/iwgtk/iwgtk-0.6-r2.ebuild

```diff
@@ -27,6 +27,10 @@ RDEPEND="
 	>=net-wireless/iwd-1.28
 "
 
+PATCHES=(
+	"${FILESDIR}"/0001-Check-if-required-icons-are-available.patch
```

Comment:
with this cherry-pick we could also just wait for the next upstream release, which should be coming soon

https://github.com/J-Lentz/iwgtk/issues/30

Reply:
but if we simply took it ... we would not need a gentoo bug to remember the icon issue, but maybe such a bug is wanted anyways or the release will come before we know it